code/core/src/main/java/com/adobe/marketing/mobile/internal/configuration/ConfigurationDownloader.kt

```diff
@@ -109,6 +109,7 @@ internal class ConfigurationDownloader {
 
         val networkCallback = NetworkCallback { response: HttpConnecting ->
             val config = handleDownloadResponse(url, response)
+            response.close()
```

Comment:
response can be null and currently handleDownloadResponse does not also handle null case. Can you fix this callback overall to handle null case?